includes/class-wc-stripe-webhook-handler.php

```diff
@@ -615,11 +615,11 @@ public function process_webhook_charge_failed( $notification ) {
 	 * @param object $notification
 	 */
 	public function process_webhook_source_canceled( $notification ) {
-		$order = WC_Stripe_Helper::get_order_by_charge_id( $notification->data->object->id );
+		$order = WC_Stripe_Order::get_by_charge_id( $notification->data->object->id );
 
 		// If can't find order by charge ID, try source ID.
 		if ( ! $order ) {
-			$order = WC_Stripe_Helper::get_order_by_source_id( $notification->data->object->id );
+			$order = WC_Stripe_Order::get_by_id( $notification->data->object->id );
```

Comment:
```suggestion
			$order = WC_Stripe_Order::get_by_source_id( $notification->data->object->id );
```

I think this is the correct function.